wasm/src/tests.rs

```diff
@@ -504,17 +504,20 @@ fn test_doc_gen() {
     let docs = qsc_doc_gen::generate_docs::generate_docs(None, None, None);
     assert!(docs.len() > 100);
     for (name, metadata, contents) in docs {
-        // filename will be something like "Microsoft.Quantum.Canon/ApplyToEachC.md"
```

Comment:
You may want to update this:
https://github.com/microsoft/qsharp/blob/6f376c48d82466ea8a50439a30432ba77d1bf5c3/playground/src/docs.tsx#L23